src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,75 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final String DEFAULT_COLOR = "White";
+    private static final int RANDOM_NUMBER = new Random().nextInt(5);

```

Comment:
in these case you will return same figure every time. Move this variable to getRandomFigure and create constant for 5